{"name": "邮箱重复时返回错误信息", "status": "passed", "attachments": [{"name": "log", "source": "4d1d28a1-2bd4-4a1f-ba8b-3950b5afdf1e-attachment.txt", "type": "text/plain"}], "start": 1741927422414, "stop": 1741927428701, "uuid": "07d25aaa-fdad-4d36-9134-80763c01324d", "historyId": "2cb599d31c902407a349da57407b7973", "testCaseId": "e9e7c8aee46571bad0866d69dd275e54", "fullName": "test_case.test_customer.TestCustomer#test_update_customer_email", "labels": [{"name": "feature", "value": "用户中心"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_customer"}, {"name": "subSuite", "value": "TestCustomer"}, {"name": "host", "value": "bindeMacBook-Pro.local"}, {"name": "thread", "value": "4581-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_customer"}]}